www/src/components/CodeBlock.tsx

```diff
@@ -64,14 +69,16 @@ function CollapsibleLiveEditor({ children, clickToCopy, handleCodeChange }: Coll
     const headingElement = getCodeBlockHeading(e.target);
 
     if (!headingElement) {
-      global.analytics.track(`openedx.paragon.docs.ui.example-code-block.${collapseIsOpen ? 'closed' : 'opened'}`, {
+      sendUserAnalyticsEvent(collapseIsOpen
+        ? EXAMPLE_CODE_BLOCK_WITHOUT_HEADING_EVENTS.CLOSED : EXAMPLE_CODE_BLOCK_WITHOUT_HEADING_EVENTS.OPENED, {
         value: `${componentNameAndCategory}id-not-generated`,
```

Comment:
can you extract event definition outside of function arguments? it's very hard to read right now, maybe change it to something like:

```js
const event = collapseIsOpen
  ? EXAMPLE_CODE_BLOCK_WITHOUT_HEADING_EVENTS.CLOSED
  : EXAMPLE_CODE_BLOCK_WITHOUT_HEADING_EVENTS.OPENED;

sendUserAnalyticsEvent(event, { value: `${componentNameAndCategory}id-not-generated`});
```

and do the same for the event below

Reply:
Done